### ISSUE TITLE
Screen readers (Voice over) do not announce data points when the chart is resized and made smaller 

### ISSUE THREAD

**Issue by nischayv** [highcharts/highcharts#15103]
#### Expected behaviour

When there are lots of data points and the chart is resized to be made smaller, the data point dots are no longer visible and the chart becomes a smooth line. On hovering over the line the current data point is displayed along with a tooltip (see screenshots). This same behavior works when using keyboard navigation. But unlike larger screens, each data point is no longer announced using the screen reader on smaller screens.

#### Actual behaviour

When navigating through the data points on smaller screens, the data points should be announced by the screen reader.

#### Product version
Highcharts  7.0.0

#### Affected browser(s)
Chrome

#### Screenshots
<img width="1296" alt="Screen Shot 2021-02-09 at 11 27 54 AM" src="https://user-images.githubusercontent.com/13353147/107545866-61318f80-6b91-11eb-88e8-c8e44946cdcf.png">
<img width="424" alt="Screen Shot 2021-02-09 at 11 27 43 AM" src="https://user-images.githubusercontent.com/13353147/107545893-68589d80-6b91-11eb-9669-6c839d252834.png">

**Comment by PaulDalek** [comment]
Hi @nischayv 

Thank you for reporting this issue.

FYI @oysteinmoseng

**Comment by KacperMadej** [comment]
@nischayv 

Please add a live demo for the problem and try with a newer version of Highcharts - the current is 9.0.0.
